detection-rules/link_fake_password_expiration.yml

```diff
@@ -22,22 +22,38 @@ source: |
   // body contains expire, expiration, loose, lose 
   and (
     regex.icontains(body.current_thread.text,
-                    '(expir(e)?(ation|s)|\blo(o)?se\b|office 365|re.{0,3}confirm)'
+                    '(expir(e)?(ation|s)|\blo(o)?se\b|office 365|re.{0,3}confirm)|due for update'
     )
     and not strings.icontains(body.current_thread.text, 'link expires in ')
   )
   
-  // subject or body contains account or access
-  and any([subject.subject, body.current_thread.text],
-          regex.icontains(body.current_thread.text, "account|access|your email")
+  and (
+    // subject or body contains account or access
+    any([subject.subject, body.current_thread.text],
+        regex.icontains(body.current_thread.text, "account|access|your email")
+    )
+    // suspicious use of recipients email address
+    or any(recipients.to,
+           strings.icontains(strings.replace_confusables(subject.subject),
+                             .email.local_part
+           )
+    )
+    or any(recipients.to,
```

Comment:
does this work to avoid looping through `recipients.to` a second time? 
```
  // suspicious use of recipients email address
  or any(recipients.to,
         strings.icontains(strings.replace_confusables(subject.subject),
                           .email.local_part
         )
         or strings.icontains(strings.replace_confusables(body.current_thread.text
                              ),
                              .email.email
         )
  )
  ```

Reply:
sure does, will add, ty!